fala/assets-src/scripts/cookies.js

```diff
@@ -0,0 +1,70 @@
+const cookieBanner = document.getElementById("cookie-banner");
+const defaultMessage = document.getElementById("default-message");
+const acceptedMessage = document.getElementById("accepted-message");
+const rejectedMessage = document.getElementById("rejected-message");
+
+// Set cookies
+const setCookie = (name, value, days) => {
+  let expires = "";
+  if (days) {
+    const date = new Date();
+    date.setTime(date.getTime() + (days * 24 * 60 * 60 * 1000));
+    expires = ";expires=" + date.toUTCString();
+  }
+  document.cookie = `${name}=${value}${expires};path=/;samesite=lax`;
+}
+
+// Accept analytics cookies
+const acceptCookieBanner = document.getElementById("accept-cookies");
+if (acceptCookieBanner) {
+  acceptCookieBanner.addEventListener("click", () => {
+    setCookie("cookiePermission", "Allowed", 30);
+    defaultMessage.hidden = true;
+    acceptedMessage.hidden = false;
+  })
+};
+
+// Reject analytics cookies
+const rejectCookieBanner = document.getElementById("reject-cookies");
+if (rejectCookieBanner) {
+  rejectCookieBanner.addEventListener("click", () => {
+    setCookie("cookiePermission", "Rejected", 30);
+    defaultMessage.hidden = true;
+    rejectedMessage.hidden = false;
+  })
+};
+
+// Hide accepted message
+const hideCoookieAcceptMessage = document.getElementById("accepted-hide");
+if (hideCoookieAcceptMessage) {
+  hideCoookieAcceptMessage.addEventListener("click", () => {
+    acceptedMessage.hidden = true;
+    cookieBanner.hidden = true;
+  })
+};
+
+// Hide rejected message
+const hideCoookieRejecttMessage = document.getElementById("rejected-hide");
+if (hideCoookieRejecttMessage) {
+  hideCoookieRejecttMessage.addEventListener("click", () => {
+    rejectedMessage.hidden = true;
+    cookieBanner.hidden = true;
+  })
+};
+
+// Cookie policy form
+const cookiePolicyForm = document.getElementById("cookie_policy_page_choice");
+if (cookiePolicyForm) {
+  cookiePolicyForm.addEventListener("submit", (e) => {
+    e.preventDefault();
+    const selectedValue = document.querySelector("input[name='cookies']:checked").value;
+    if (selectedValue === "Allowed") {
+      setCookie("cookiePermission", "Allowed", 30);
+      tagMan();
+      location.reload();
```

Comment:
I don't think we should refresh the users browser in the cookie policy page. We don't do this for the banner so why do it for the cookie policy page? 
It's an unnecessary refresh and HTTP request when the user will do that themselves naturally when they navigate away, which is a useful HTTP request.

Reply:
Actually @psweeting1 I can do this in the JavaScript. I'll remove reload from FALA.